python/empyrical_evidences/README.md

```diff
@@ -0,0 +1,117 @@
+<h1 align="center">Empyrical Evidences</h1>
+
+<p align="center">
+  Search hackernews for comment relevant to academic papers.
+  Powered by Together.ai and DBOS.
+</p>
+
+## What does the app do
+
+This application uses together.ai inference API to identity relevant topics in academic papers and rank the most relevant related comments from hackernews. It is orchestrated with DBOS Transact and demonstrates how DBOS:
```

Comment:
What It Does

How It Works

- You upload papers and it embeddings
- You search with multi-agent workflow

What DBOS Does

Instructions + How To Use It